test/test_ensembling_pipelines_generation.py

```diff
@@ -114,8 +114,9 @@ def test_experimenter_can_generate_same_models(self):
 
         primitives = primitive_list_from_pipeline_json(generated_pipelines["classification"][0].to_json_structure())
         primitives_used = set(primitives)
-        primitives_repeated = 16  # preprocessor repeats 5 times, model repeats 5 times, concat repeats 2 times and
-                                 # construct predictiond repeats 5 times
+        # extract by semantic types repeats 2 times, preprocessor repeats 5 times, model repeats 5 times,
+        # concat repeats 2 times, and construct predictions repeats 5 times
+        primitives_repeated = 17                     
```

Comment:
These test updates were needed because I added a step to extract attributes by semantic type to all pipelines.